src/idd/idd.base.js

```diff
@@ -1428,67 +1465,87 @@ var _initializeInteractiveDataDisplay = function () { // determines settings dep
             return plot;
         };
 
-        //------------------------------------------------------------------------------
-        //Navigation
-        if (_isMaster) {
-            //Initializing navigation
-            var _navigation = new InteractiveDataDisplay.Navigation(this, setVisibleRegion);
+            //------------------------------------------------------------------------------
+            //Navigation
+            if (_isMaster) {
+                //Initializing navigation
+                var _navigation = new InteractiveDataDisplay.Navigation(this, setVisibleRegion);
         }
 
-        Object.defineProperty(this, "navigation", { get: function () { if (_isMaster) return _navigation; else return _master.navigation; }, configurable: false });
+        Object.defineProperty(this, "navigation", { get: function () { if (_isMaster) return _navigation; else return _master.navigation; }, configurable: false
+        });
```

Comment:
@ElenaPochernina Something strange happens with indentation; please fix the indentation of the file.